README.md

```diff
@@ -1,56 +1,43 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
+## Morse Decoder
 
+> It is an encryptor to decode morse messages. 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- Ruby
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
+1- Copy this link `https://github.com/HSMathebula/morse-decoder.git` <br>
+2- Get the directory that you want to clone the repository. <br>
+3- Open the command prompt in this directory. <br>
+4- Write `git clone https://github.com/HSMathebula/morse-decoder.git` <br>
+5- Go to the repository folder in your command prompt `cd morse-decoder`. <br>
 
-### Run tests
+### Prerequisites
 
-### Deployment
+IRB or VSCode
 
```

Comment:
- [ ] OPTIONAL
Kindly add instruction in the ReadMe on how to run your app. This will make your ReadMe appear more professional.

Reply:
This issue is well resolved! ✔️